src/cswinrt/code_writers.h

```diff
@@ -7412,6 +7412,7 @@ internal static global::System.Guid IID { get; } = new Guid(new byte[] { % });
 internal volatile static bool _RcwHelperInitialized;
 unsafe static @Methods()
 {
+ComWrappersSupport.RegisterHelperType(typeof(%), typeof(%));
```

Comment:
Out of curiosity, does this entirely replaces using `[WinRTHelperType]` on new projections? 🤔

Reply:
At least as it is right now, no.  This only does it for generic interfaces.